src/main/scala/com/microsoft/partnercatalyst/fortis/spark/StreamsChangeListener.scala

```diff
@@ -103,11 +103,11 @@ object StreamsChangeListener {
     }
   }
 
-  private[spark] class ContextStopRunnable(settings: FortisSettings, ssc: StreamingContext, scheduler: ExecutorService) extends Runnable with Loggable {
+  private[spark] class ContextStopRunnable(settings: FortisSettings, ssc: StreamingContext, scheduler: ScheduledExecutorService) extends Runnable with Loggable {
     override def run(): Unit = {
       StreamsChangeListener.suggestedExitCode = 10
       System.err.println(s"Requesting streaming context stop now.")
-      val timeoutTask = scheduler.submit(new TimeoutRunnable)
+      val timeoutTask = scheduler.schedule(new TimeoutRunnable(), 30L, TimeUnit.SECONDS)
```

Comment:
Should the `30L` live in [Constants.scala](https://github.com/jcjimenez/project-fortis-spark/blob/42b3c9f7193c4fd9830e19e7ae133f876b5a1638/src/main/scala/com/microsoft/partnercatalyst/fortis/spark/Constants.scala)?